packages/client/src/components/app/forms/RelationshipField.svelte

```diff
@@ -105,19 +105,29 @@
     }
   }
 
-  $: fetchRows(searchTerm, primaryDisplay)
+  $: fetchRows(searchTerm, primaryDisplay, defaultValue)
```

Comment:
This `defaultValue` can now be removed, right?

Reply:
This is used for reactivity - I want to trick the `fetchRows` if the `defaultValue` changes for any reason.